src/models/room.ts

```diff
@@ -1613,11 +1644,38 @@ export class Room extends ReadReceipt<EmittedEvents, RoomEventHandlerMap> {
 
     public threadsReady = false;
```

Comment:
Do you know why this is public, ooi?

Reply:
I actually do not, but from what my IDE can tell me, neither matrix-js-sdk (outside of room.ts) nor matrix-react-sdk use this variable, so it might as well be private.